telemetry/README.md

```diff
@@ -16,8 +18,8 @@ Make changes in that file, then from the AWS Toolkit repo root run:
 To update the generated output. When you are satisfied with generated telemetry, move
 the definitions you just added to the appropriate file in `definitions` in this repo.
 
-
 #### JetBrains
+
```

Comment:
Do we need appropriate ### Visual Studio sections?

Reply:
It looks like some of this material is written up in csharp/README.md -- please let me know if you have strong feelings about relocating some of that into here, or if a simple pointer is sufficient